packaging/MacSDK/packaging/resources/postinstall

```diff
@@ -60,16 +60,23 @@ if [ -d ${FW_CURRENT} ]; then
     cd ${FW_CURRENT}/etc
     # Make sure we run the files we lay down, and not other stuff installed on the system
     export PATH="${FW_CURRENT}/bin:$PATH"
+    LOG_DIR=/Library/Logs/Mono.framework
```

Comment:
pretty sure you need to `mkdir` the folder before you can use it.

Reply:
🤦‍♂